articles/app-service/app-service-hybrid-connections.md

```diff
@@ -22,160 +22,165 @@ ms.author: ccompy
+- TCP access to the Hybrid Connection endpoint
+- Ability to do DNS look-ups on the endpoint host and the Azure Service Bus namespace
 
-The HCM supports both new hybrid connections as well as the older BizTalk hybrid connections.
+The HCM supports both new Hybrid Connections, as well as the older BizTalk Hybrid Connections.
+
+> [!NOTE]
+> Azure Relay relies on Web Sockets for connectivity. Because Windows Server 2008 R2 does not support Web Sockets, you cannot use the new Hybrid Connections if the HCM is installed on Windows Server 2008 R2. You will need to install the HCM on Windows Server 2012 or later if you want to use Hybrid Connections that use Azure Relay.
+>
+>
 
 ### Redundancy ###
 
-Each HCM can support multiple hybrid connections.  Also, any given hybrid connection can be supported by multiple HCMs.  The default behavior is to round robin traffic across the configured HCMs for any given endpoint.  If you want high availability on your hybrid connections from your network, simply instantiate multiple HCMs on separate machines. 
+Each HCM can support multiple Hybrid Connections.  Also, any given Hybrid Connection can be supported by multiple HCMs.  The default behavior is to round robin traffic across the configured HCMs for any given endpoint.  If you want high availability on your Hybrid Connections from your network, simply instantiate multiple HCMs on separate machines. 
 
-### Manually adding a hybrid connection ###
+### Manually adding a Hybrid Connection ###
 
-If you wish somebody outside of your subscription to host an HCM instance for a given hybrid connection, you can share with them the gateway connection string for the hybrid connection.  You can see this in the properties for a hybrid connection in the [Azure portal][portal]. To use that string, click the **Configure manually** button in the HCM and paste in the gateway connection string.
+If you wish somebody outside of your subscription to host an HCM instance for a given Hybrid Connection, you can share the gateway connection string for the Hybrid Connection with them.  You can see this in the properties for a Hybrid Connection in the [Azure portal][portal]. To use that string, click the **Enter Manually** button in the HCM and paste in the gateway connection string.
 
 
 ## Troubleshooting ##
 
-When your hybrid connection is set up with a running application and there is at least one HCM that has that hybrid connection configured, then the status will say **Connected** in the portal.  If it does not say **Connected** then it means that either your app is down or your HCM cannot connect out to Azure on ports 80 or 443.  
+When your Hybrid Connection is set up with a running application and there is at least one HCM configured with that Hybrid Connection, the status will say **Connected** in the portal.  If it does not say **Connected**, either your app is down or your HCM cannot connect out to Azure on ports 80 or 443.  
```

Comment:
When your Hybrid Connection is set up and there is at least one HCM configured with that Hybrid Connection, the status will say **Connected** in the portal.  If it does not say **Connected**, then your Hybrid Connection is not configured with any HCM that has access to Azure.